lib/info-extras.js

```diff
@@ -124,11 +124,49 @@ exports.getPublished = (body) => {
  * @return {Array.<Object>}
  */
 exports.getRelatedVideos = (body) => {
-  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': {"rvs":', '},');
+  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': ', ',\n');
+  let watchNextJson;
   try {
     jsonStr = JSON.parse(jsonStr);
-  } catch (err) {
+    watchNextJson = JSON.parse(jsonStr.watch_next_response);
+  }
+  catch (err) {
     return [];
   }
-  return jsonStr.split(',').map((link) => qs.parse(link));
-};
+  let rvsParams = jsonStr.rvs.split(',').map((e) => {
+    return {
+      id: util.between(e, 'id=', '&'),
+      shortViewCount: util.between(e, 'short_view_count_text=', '+views')
+    };
+  })
```

Comment:
use `qs.parse()` like before. also, missing a semicolon

actually, why not keep using these parameters, it looks like they contain everything except `ucid`, not sure what this param is either. @omarroth what do you think?

Reply:
ucid contains the channel id @fent

Reply:
also, view count is not present.

Reply:
> use `qs.parse()` like before. also, missing a semicolon
is it necessary to use qs.parse()

Reply:
there's already a library that parses querystrings, why try to parse out the values yourself?

Reply:
👍